Cargo.toml

```diff
@@ -1,9 +1,11 @@
 [workspace]
 members = [
+    "crates/gitbutler-analytics",
```

Comment:
The `analytics` crate was a side-effect of making `watcher` standalone, which led to further separating dependencies. Added dependencies that are only used in one module is definitely a good motivation for a separate crate, even if otherwise it might not seem to be quite worth it.